src/lib/template/dom-bind.html

```diff
@@ -78,6 +78,9 @@
     _template: null,
 
     created: function() {
+      if (Polymer.Settings.strictTemplatePolicy) {
```

Comment:
Test this throws as expected using the custom elements polyfill (and doesn't halt other elements from booting up)

Reply:
Confirmed that on CE polyfill (both v0 and v1) element upgrading halts when a constructor throws.

In V1 it actually try/catch'es, but [re-throws the error](https://github.com/webcomponents/custom-elements/blob/master/src/CustomElementInternals.js#L285).  Not sure why.  Filed issue here: https://github.com/webcomponents/custom-elements/issues/176